docs/conf_sso_test_msscolab.rst

```diff
@@ -52,15 +52,15 @@ Generate metadata files
 
 This involves generating necessary metadata files for both the identity provider and the service provider. You can generate them by simply running the appropriate command.
 
-Before executing this, you should set `USE_SAML2=True` as described in the third step(USE_SAML2).
+Before executing this, you should set `USE_SAML2=True` as described in the third step(Enable USE_SAML2).
 
 `$ python mslib/mscolab/mscolab.py sso_conf --init_sso_metadata`
 
 
 Start Identity provider
 -----------------------
 
-Once you setted certificates and metada files you can start mscolab server and local identity provider. To start local identity provider, simpy execute
+Once you set certificates and metada files you can start mscolab server and local identity provider. To start local identity provider, simply execute
 
 `$ python mslib/idp/idp.py idp_conf`
 
```

Comment:
the script needs to become added to the entrypoints e.g. msidp = ...

https://github.com/Open-MSS/MSS/blob/GSOC2023-NilupulManodya/localbuild/meta.yaml#L15

Reply:
just thinking more on this, this is only for testing as long as we won't add this to the "existing" user database.

When it is for tests a user command makes not soo much sense. And also the location as a "main" module  below mslib is not that good. 
- we should think about making it a provider for the existing mscolab databae
- or move it into the _test_utils section